### PR TITLE
list mode

### PR DESCRIPTION
**Pull request by sbcgua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,12 @@ lo_map->set(
   iv_val = '2' ). " raises cx_no_check
 ```
 
+- you may switch the map into "list" mode - this way there is no guarantee for the record uniquiness, but can be usedful for some applications.
```
Comment (mbtools) [inline README.md:147]: uniqueness, useful

File: changelog.txt [hunk 2]
```diff
@@ -8,6 +8,10 @@ Legend
 + : added
 - : removed
 
+v1.0.5, 2024-??-??
```
Comment (mbtools) [inline changelog.txt:11]: 2024-11-27

**Comment by mbtools** [comment]
Nice addition 👍

File: README.md (hunk 1 above)
Comment (sbcgua) [reply README.md:147]: tnx! fixed

File: changelog.txt (hunk 2 above)
Comment (sbcgua) [reply changelog.txt:11]: I usually don't bump version for minor changes. But on the other hand there are so few changes to the string map, that why not. Done.

**Pull request merged**